dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -309,11 +309,25 @@ object LagomPlugin extends AutoPlugin {
     val lagomKafkaPort = settingKey[Int]("Port used by the local kafka broker")
     val lagomKafkaAddress = settingKey[String]("Address of the kafka brokers (comma-separated list)")
 
-    /** Allows to integrate an external Lagom project in the current build, so that when runAll is run, this service is also started.*/
+    @deprecated("Use lagomExternalJavadslProject or lagomExternalScaladslProject instead", "1.3.0")
     def lagomExternalProject(name: String, module: ModuleID): Project =
```

Comment:
Since Scala support is new, would it make sense for `lagomExternalProject` to call `lagomExternalJavadslProject`? Then it would just start working if there are any existing users of the now-deprecated API.

Also, the original issue mentioned that it's been broken since 1.2. That change could be backported to 1.2.x to fix it there.